src/test/java/org/heigit/ohsome/ohsomeapi/controller/DataExtractionTest.java

```diff
@@ -446,7 +458,19 @@ public void contributionsGeometryCoordinateTest() {
         + "&filter=id:node/3429511451&time=2017-01-01,2019-01-01", JsonNode.class);
     JsonNode feature = Helper.getFeatureByIdentifier(response, "@osmId", "node/3429511451");
     assertEquals(49.418466, feature.get("geometry").get("coordinates").get(1).asDouble(), 0);
-  }
+    
+  }
+    @Test
+    public void contributionTypesTest() {
```

Comment:
Only one thing I have to point out: as it's very likely that we will have a `/groupBy/contributionType` endpoint at some point, it would be better to make it clear in the name of this test that it's about the parameter `properties=contributionTypes`.

```suggestion
    public void contributionTypesPropertiesParameterTest() {
```

Reply:
Yes, you are right!